subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -189,6 +189,11 @@ static MFIFO_DEFINE(pdu_rx_free, sizeof(void *), PDU_RX_CNT);
 
 #define PDU_RX_POOL_SIZE (PDU_RX_NODE_POOL_ELEMENT_SIZE * \
 			  (RX_CNT + BT_CTLR_MAX_CONNECTABLE))
+#if defined(CONFIG_SOC_OPENISA_RV32M1_RISCV32)
+/* ULL support for Rx buffer overflow */
+#define LL_PDU_SIZE (PDU_RX_NODE_POOL_ELEMENT_SIZE - NODE_RX_STRUCT_OVERHEAD)
+u32_t ll_pdu_size = LL_PDU_SIZE;
```

Comment:
why not have this define in vendor hal headers, and why the global variable use?

Reply:
It's used in radio.c, where rx buffer size is checked against PDU size

Reply:
A function in ULL returning the Rx buffer size is better?

Reply:
can you use the max_len passed in radio_pkt_configure call? then you dont need this global.

Reply:
done